core/common/src/main/java/alluxio/grpc/GrpcManagedChannelPool.java

```diff
@@ -231,6 +232,7 @@ private ManagedChannel createManagedChannel(ChannelKey channelKey) {
     if (channelKey.mPlain) {
       channelBuilder.usePlaintext();
     }
+    channelBuilder.executor(ForkJoinPool.commonPool());
```

Comment:
Have you done any experiment on what the impact is to streaming RPCs in data path?